project_start/src/App.js

```diff
@@ -62,11 +64,12 @@ function App() {
             <header className="App-header">
               <a
                 className="App-link"
-                href="http://localhost:8888/login"
+                href={link}
```

Comment:
consider moving 'http://localhost:8888' to a constant variable or a shared js file